config/routes.rb

```diff
@@ -10,6 +10,7 @@
   root to: redirect("/start")
 
   mount Avo::Engine, at: Avo.configuration.root_path
+  mount GoodJob::Engine => "/good-job"
```

Comment:
This probably needs proper auth, and Avo as well should be accessible only with an admin user.